benchmark/performance_utils.py

```diff
@@ -372,7 +371,7 @@ def run(self):
                 shape_desc=self.shape_desc,
             )
             print(attri)
-            logging.info(attri.to_dict())
+            recordLogger.info(attri.to_dict())
```

Comment:
Sorry, I don't get time to go through all 374 lines above. 
@kiddyjinjin , do we tested `logger = logging.getLogger("flag_gems.benchmark.record")` before?

My question: as default logger(`logging.getLogger`) is able to log file by `enable_gems(....)`, and in this file, which using logging without getLogger seems wrong.

Back to `Since this PR focuses on fixing the missing log file issue...`. Which means, we confirmed and tested the log file will missing in benchmark testing, when both `enable_gems(....)` and `logging.getLogger` configured by design?